srcpkgs/kdenlive/template

```diff
@@ -9,16 +9,16 @@ hostmakedepends="
 makedepends="
  kdeclarative-devel kfilemetadata5-devel knewstuff-devel knotifyconfig-devel
  kplotting-devel mlt-devel qt5-multimedia-devel qt5-webkit-devel purpose-devel
- v4l-utils-devel ksolid-devel qt5-quickcontrols2-devel"
+ v4l-utils-devel ksolid-devel qt5-quickcontrols2-devel qt5-networkauth-devel"
 depends="breeze-icons dvdauthor ffmpeg frei0r-plugins kinit qt5-quickcontrols
- qt5-quickcontrols2 vlc"
+ qt5-quickcontrols2 qt5-networkauth vlc"
```

Comment:
This looks outdated... `qt5-quickcontrols2` and `qt5-networkauth` are detected automatically:

```
   SONAME: libQt5NetworkAuth.so.5 <-> qt5-networkauth>=5.15.2_3
```

And `qt5-quickcontrols` is probably not necessary, given that it uses quickcontrols2...

Reply:
No, qt5-quickcontrols is required. kdenlive won't start if it is not present on the system. Please do not drop dependencies without checking if they are needed.

Reply:
Ok, so we only remove `quickcontrols2` and `networkauth`

Reply:
> Ok, so we only remove `quickcontrols2` and `networkauth`

ok. i'ill keep `qt5-quickcontrols` and remove `qt5-quickcontrols2` and `qt5-networkauth`